docs/index.html

```diff
@@ -0,0 +1,657 @@
+<!DOCTYPE html>
+<html>
+<head>
+    <link rel="stylesheet" href="main.css">
+    <meta charset="UTF-8">
+    <title>ambianic-functions-collection</title>
+                                                         </head>
+<body>
+<div class="container">
+    <h1>ambianic-functions-collection</h1>
+    <p class="sw-info">
+        
+        Version: <span class="sw-info-version">a7b6cde1-7435-479a-bd1a-20ab61f9ae08</span>,     <a href="" class="sw-external-doc"></a>
+    
+    </p>
+    <p><p>A postman collection documenting the Netlify cloud functions created for use within Ambianic-PWA</p>
```

Comment:
@vickywane this statement is confusing. This is a document for the public Ambianic Premium Subscription API. No need to mix in implementation details such as postman and netlify.

Reply:
It has been rephrased to "This collection contains a specification for all API endpoints within Ambianic's Cloud API."